packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -673,7 +693,8 @@ export class MgtPersonCard extends MgtTemplatedComponent {
       <div class="section-nav">
         ${sectionNavTemplate}
       </div>
-      <div class="section-host" @wheel=${(e: WheelEvent) => this.handleSectionScroll(e)} tabindex=0>
+      <div class="section-host ${this._smallView ? 'small' : ''}" @wheel=${(e: WheelEvent) =>
```

Comment:
_nit:_ can we move to one attribute per line for readability when we need to split lines?